docs/buildkit/tutorial_3d_platformer.md

```diff
@@ -0,0 +1,1337 @@
+# Tutorial: 3D Platformer
+
+In today's tutorial we're going to develop a 3D platformer game which can be played in your web browser. This game will leverage the Hathora `BuildKit` client and server libraries to facilitate multiplayer functionality.
+
+Note that this tutorial is aimed at developers who may not have experience with Hathora, TypeScript, or even multiplayer game development in general. If you're a more advanced developer, feel free to skip sections so long as you feel confident in your understanding of what's going on.
+
+## Repo
+
+The source code of the 3D platformer can be located [here](https://github.com/hathora/3d-platformer). The repo's README file will guide you through running the game on your local machine, and there is an existing build of the game deployed to the web which can be sampled [here](https://3d-platformer.netlify.app/).
+
+Upon completion of this tutorial, your game will look like this:
+
+![A screenshot of the completed 3D platformer demo game, running fullscreen in the browser.](https://user-images.githubusercontent.com/7004280/213311928-18d0ed21-d309-4a30-8c7c-36cf0655d0f5.png)
```

Comment:
can this screenshot include another player to indicate that it's multiplayer?

Reply:
Updated to include 2 additional players in the viewport, from a similar angle.